lib/user.rb

```diff
@@ -0,0 +1,32 @@
+require 'bcrypt'
+require_relative './database_connection'
+
+class User
+  def self.create(name:, username:, email:, password:)
+    encrypted_password = BCrypt::Password.create(password)
+
+    result = DatabaseConnection.query("INSERT INTO users \
+      (name, username, email, password) \
+      VALUES('#{name}', '#{username}', '#{email}', '#{encrypted_password}') \
+      RETURNING id, name, username, email;")
```

Comment:
Well done to split these over multiple lines - I think it would be easier to read if they were justified though.